src/main/java/frc/robot/constants/KitBot2022Constants.java

```diff
@@ -65,27 +65,27 @@ public boolean driveMotorRightFollowerInverted() {
     }
 
     @Override
-    public boolean hasClimber2020() {
-        return false;
+    public boolean hasClimber2022() {
+        return true;
```

Comment:
kitbot does not have a climber
```suggestion
        return false;
```